spark/src/main/scala/org/apache/comet/CometSparkSessionExtensions.scala

```diff
@@ -438,7 +438,7 @@ class CometSparkSessionExtensions
               op
           }
 
-        case op: LocalLimitExec if getOffset(op) == 0 =>
+        case op: LocalLimitExec =>
```

Comment:
Did the logic change here? Shouldn't there be an `if op.offset == 0` match guard here?

Reply:
`LocalLimitExec` does not have `offset` to begin with (maybe it was different in older versions). So `getOffset` was returning `0` always